wiring/inc/spark_wiring_system.h

```diff
@@ -32,7 +32,6 @@
 #include "system_sleep.h"
 #include "system_cloud.h"
 #include "system_event.h"
-#include "core_hal.h"
```

Comment:
`core_hal.h` is not about Core platform. This header needs to be kept.

Reply:
it's there again two lines lower

Reply:
:see_no_evil: